src/renderer/views/Playlist/Playlist.js

```diff
@@ -36,6 +35,9 @@ export default Vue.extend({
     },
     currentInvidiousInstance: function () {
       return this.$store.getters.getCurrentInvidiousInstance
+    },
+    currentLocale: function () {
+      return this.$store.getters.getCurrentLocale.replace('_', '-')
```

Comment:
Maybe include
import i18n from '../../i18n/index'
at the top
and then get the current locale with:
i18n.locale